hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/TestRatisOverReplicationHandler.java

```diff
@@ -154,6 +155,7 @@ public void testOverReplicatedQuasiClosedContainerWithDifferentOrigins()
    * prioritize creating delete commands for unhealthy replicas over quasi
    * closed replicas.
    */
+  @Ignore("HDDS-7804")
```

Comment:
If this test is no longer valid based on the changes here, probably best to remove it rather than just ignore it.

Reply:
Yeah, I think we need a similar test later on but in a different context. I've removed it for now.